eng/pipelines/templates/Build_and_UnitTest.yml

```diff
@@ -338,25 +338,19 @@ steps:
     ArtifactType: Container
   condition: "and(succeeded(), eq(variables['BuildRTM'], 'false'), eq(variables['IsOfficialBuild'], 'true'))"
 
-- task: PowerShell@1
-  displayName: 'OptProfV2:  set the TestDrop environment variable'
-  inputs:
-    scriptType: 'inlineScript'
-    inlineScript: |
-      [string] $bootstrapperInfoFilePath = "$Env:BUILD_STAGINGDIRECTORY\MicroBuild\Output\BootstrapperInfo.json"
-      $json = Get-Content $bootstrapperInfoFilePath | ConvertFrom-Json
-      [string] $buildDropPath = $json.BuildDrop
-      Write-Host "Build drop:  $buildDropPath"
-      [string] $testDropPath = $buildDropPath.Replace('/Products/', '/Tests/').Substring('https://vsdrop.corp.microsoft.com/file/v1/'.Length)
-      Write-Host "Test drop:  $testDropPath"
-      Write-Host "##vso[task.setvariable variable=TestDrop]$testDropPath"
-  condition: "and(succeeded(), eq(variables['BuildRTM'], 'false'), eq(variables['IsOfficialBuild'], 'true'))"
-
 - task: MSBuild@1
   displayName: 'OptProfV2:  generate a .runsettings file'
   inputs:
     solution: 'build\NuGet.OptProfV2.runsettingsproj'
-    msbuildArguments: '/p:OutputPath="$(Build.Repository.LocalPath)\artifacts\RunSettings" /p:TestDrop="$(TestDrop)" /p:ProfilingInputsDrop="ProfilingInputs/$(System.TeamProject)/$(Build.Repository.Name)/$(Build.SourceBranchName)/$(Build.BuildId)"'
+    msbuildArguments: '/p:OutputPath="$(Build.Repository.LocalPath)\artifacts\RunSettings" /p:TestDrop="RunSettings/$(System.TeamProject)/$(Build.Repository.Name)/$(Build.SourceBranchName)/$(Build.BuildId)" /p:ProfilingInputsDrop="ProfilingInputs/$(System.TeamProject)/$(Build.Repository.Name)/$(Build.SourceBranchName)/$(Build.BuildId)" /p:IncludeProfilingInputs=$(IsOfficialBuild)'
```

Comment:
I don't see any references to `IncludeProfilingInputs` property in https://github.com/NuGet/NuGet.Client/blob/dev/build/NuGet.OptProfV2.runsettingsproj file. I am curious to learn about how this property is used in the later steps.

Reply:
The intention was to add this only in official builds: https://github.com/NuGet/NuGet.Client/blob/4d1558091b53ce9cdabc83688d0b0d01bf0b6166/build/NuGet.OptProfV2.runsettingsproj#L35

For PR builds we won't have profiling inputs, plus PR builds won't go through real opt-prof. In fact, for PR builds, we won't use this runsettings file at all, but initially I was trying to get the test assembly to run.

This will be fixed up in my next PR when I get E2E tests working on new infrastructure.